Dockerfile

```diff
@@ -23,13 +25,21 @@ RUN cp /tmp/requirements.txt /tmp/req.bak \
   && python -m pip install --no-cache-dir --use-feature=2020-resolver -r /tmp/requirements-dev.txt
 
 # compile ext and remove temp files
+# TODO: remark for issue [revise the dockerfile #1276](https://github.com/Project-MONAI/MONAI/issues/1276)
```

Comment:
TODO: remark for issue [revise the dockerfile #1276](https://github.com/Project-MONAI/MONAI/issues/1276)